.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @jsbroks @venky-wandb @zythosec @vanpelt
+* @delivery-team
```

Comment:
```suggestion
* @wandb/delivery-team @wandb/security-team 
```

Reply:
mhm we currently only have delivery-team in aws, it add overhead and we also have tf sec checks

Reply:
This isn't for tfsec, this is to add security-team as optional reviewers. Since they are in the same line only 1 person from either team will be required for review. The only time review from both delivery and security would be required would be if we added another line for specific files or directories that security must review.

Reply:
Are you sure that is how code owners work?

If the code owners are not on the same line, the pattern matches only the last mentioned code owner